tutorial.md

```diff
@@ -443,12 +453,10 @@ around in the widget tree.
 Useful debugging tools
 ----------------------
 
-This is a quick way to dump the entire widget tree to the console.
-This can be quite useful in figuring out exactly what is going on when
-working with the widgets system. For this to work, you have to have
-launched your app with `runApp()`.
+This is a quick way to dump the entire widget tree to the console. This can be
+quite useful in figuring out exactly what is going on when working with the
+widgets system.
 
-<!-- skip -->
 ```dart
-debugDumpApp();
+void dumpWidgetTree() => debugDumpApp();
```

Comment:
This is a bit confusing.  Maybe we should just keep the "skip" or remove the "dart".